### PR TITLE
Release 10.4.1

### PR DESCRIPTION
**Pull request by fxthomas**
This is for releasing 10.4.1, with two important user-visible bugfixes:
- Last song in a play queue no longer repeats (#1254)
- Add database support for MariaDB (#1188) (@muff1nman This is the commit you were thinking about?)

On the developer side, it also replaces a dead Maven repository from the dependency sources, which prevented 10.4.0 sources from being built as is on new machines with no cached packages (#1277).

### CODE REVIEW

**Comment by fxthomas** [comment]
Dear Travis, here is your fix for CVEs.

**Comment by fxthomas** [comment]
Here we go again!

**Comment by fxthomas** [comment]
This should be the last of it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,17 @@
 # airsonic/airsonic
 # -->
 
+## v10.4.1 - ?
+
+Fixes:
```
Comment (jvoisin) [inline CHANGELOG.md]:
```suggestion
### Fixes
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
...
+
+- Last song in a play queue no longer repeats (#1254)
+- Add database support for MariaDB (#1188)
+
+Developer:
```
Comment (jvoisin) [inline CHANGELOG.md]:
```suggestion
###  Developer
```

File: CHANGELOG.md (hunk 1 above)
Comment (fxthomas) [reply CHANGELOG.md]: Done and done!

**Comment by fxthomas** [comment]
I'm not sure how to publish the release once it is done, but I created a draft in the Releases page with the same text I added to `CHANGELOG.md`.

**Comment by jvoisin** [comment]
Please do document the release process then :)

**Comment by muff1nman** [comment]
The release process is mostly documented at https://github.com/airsonic/airsonic/blob/master/contrib/release.md. In the interest of time I'll push this release out, but I'd be glad to see someone else try out the next release. One thing is to make sure you get access to the airsonic-passwords repo (@jvoisin has an example PR [here](https://github.com/airsonic/airsonic-passwords/pull/1)) so that you can push to docker hub.

**Comment by fxthomas** [comment]
Ah, thank you! That makes it much clearer. I'll keep that in mind for the next release ;)

**Comment by jvoisin** [comment]
Kudos for doing the release!

**Pull request merged**